com.github.marhkb.Pods.Devel.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id": "com.github.marhkb.Pods.Devel",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "pods",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--env=RUST_LOG=pods=debug",
```

Comment:
are you sure it should be that verbose?

Reply:
This is a leftover from early days. The application actually has a cli argument for this and in case it is omitted, it uses `info` for release builds and `debug` for debug builds. So, I've just removed that line.